package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "graceful-fs": "^4.2.4",
+    "lru-cache": "^10.2.2",
```

Comment:
I don't think we need this big package here, you can rewrite this package and put it inside project, because we use only the `max` value

Reply:
i'll implement it and update